dot-net-sdk/dto/bandit/Attributes.cs

```diff
@@ -1,3 +1,35 @@
+using System.Text.Json.Serialization;
+using Newtonsoft.Json;
+
 namespace eppo_sdk.dto.bandit;
 
-public record AttributeSet(IDictionary<string, double> NumericAttributes, IDictionary<string, string> CategoricalAttributes);
+public class AttributeSet
+{
+    [JsonProperty("numeric_attributes")]
```

Comment:
The `AttributeSet` type is relied upon for parsing test cases, which, for some reason depart from camelCase for these fields.